content/en/docs/reference/glossary/quantity.md

```diff
@@ -28,3 +29,8 @@ intentionally lowercase), `M` (mega), `G` (giga), `T` (terra), `P` (peta),
 
 The accepted binary (power-of-2) units are `Ki` (kibi), `Mi` (mebi), `Gi` (gibi),
 `Ti` (tebi), `Pi` (pebi), `Ei` (exbi).
+
```

Comment:
I'm wondering if we should leave linked examples out of the glossary; since then it would become more of an appendix? I could be sold either way on that...

Reply:
Your call. I think there may be links in other places. I felt like I needed more context.

Reply:
Check out entry, FlexVolume, as an example. Though, I don't feel strongly about it.